### PR TITLE
Update Part 2

### PR DESCRIPTION
**Pull request by Zzyzzzzzzzz**

### CODE REVIEW

File: Class_Group_Project_Template.Rmd [hunk 1]
```diff
@@ -122,13 +124,11 @@ df$price = parse_number(df$price)
 # Transformation (date)
 df$year = as.integer(substr(df$date, 1, 4))
 df$month = as.integer(substr(df$date, 5, 6))
-
-year = df$year
```
Comment (floralo40) [inline Class_Group_Project_Template.Rmd:126]: Can you add this line back in? Deleting it broke some later code

File: Class_Group_Project_Template.Rmd [hunk 2]
```diff
@@ -122,13 +124,11 @@ df$price = parse_number(df$price)
...
-month = df$month
```
Comment (floralo40) [inline Class_Group_Project_Template.Rmd:127]: Can you add this line back in? Deleting it broke some later code

**Review by floralo40 — CHANGES_REQUESTED** [review]
Requested 2 deleted lines of code to be added back in. Deleting it resulted in some Part V code not working.

File: Class_Group_Project_Template.Rmd (hunk 2 above)
Comment (kayleeisokay) [reply Class_Group_Project_Template.Rmd:127]: Sorry that was me lol. I needed that for the model monitoring part

**Review by floralo40 — APPROVED** [review]

**Pull request merged**